Dockerfile

```diff
@@ -10,6 +10,6 @@ CMD ["/go/bin/docker-volume-sshfs"]
 
 FROM alpine
 RUN apk update && apk add sshfs
-RUN mkdir -p /run/docker/plugins /mnt/state /mnt/volumes
+RUN mkdir -p /run/docker/plugins /mnt/state /mnt/volumes .ssh
```

Comment:
Should this be adjusted to be `/root/.ssh` instead of just `.ssh`?

Reply:
I'll take a look, I'm not sure it's needed since the tests are passing right now